src/relay/transforms/device_planner.cc

```diff
@@ -877,7 +876,7 @@ class DeviceDefaulter : public ExprVisitor {
 };
 
 /* =============== Phase 3 =============== */
-
+// TODO(@electriclilies): rewrite this comment
```

Comment:
I think just function attributes bullet needs to be updated, the rest still stands.

Reply:
done!